airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/containers.py

```diff
@@ -44,39 +62,36 @@ def build_container(
         "/src",
         dagger_client.host().directory(
             ".",
```

Comment:
Good question and I'll add a comment to explain this to legitimately mystified future readers. Mounting a volume on an existing mount point effectively overwrites it. In this case, yes, this is deliberate.

The whole `/airbyte/repo` thing originated from a previous (worse) idea I had of copying some warm-up files in `/airbyte` and then later on mounting the sources in `/airbyte/repo`. It wasn't that good of an good idea.

Reply:
Got it! And overwriting the directory doesn't mess with any binaries we've installed from poetry in the installation step that we need around to run (e.g. black)? 

Since both you and augustin ran it locally with no issues, I assume its fine.

Reply:
I hadn't thought of that, but that doesn't seem to be the case, fortunately.